src/lib/pages/interfaces.ts

```diff
@@ -88,6 +89,10 @@ export type PageMeta = Pick<
   | 'hasContent'
   | 'galleryImage'
   | 'syncWithPageId'
+  | 'autoGenerated'
```

Comment:
There should not be any reason to add these fields to PageMeta. additionalPaths is only needed when you're visiting the app directly, we shouldn't need it in the front-end.

Reply:
Sounds good, reverted this interface change